### PR TITLE
inital changes for docs regarding dyn operations

### PR DESCRIPTION
**Pull request by anushkamittal2001**
## Related issue #
https://github.com/kyverno/kyverno/pull/8437

## Proposed Changes
We want to add a note in the installation about how the operations are added in validating and mutating webhook configurations

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Comment by chipzoller** [comment]
Can you please link to the code PR which is responsible for this?

**Comment by realshuting** [comment]
> Can you please link to the code PR which is responsible for this?

Linked.

File: content/en/docs/Installation/_index.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ A standard Kyverno installation consists of a number of different components, so
 * **Webhooks**
   * ValidatingWebhookConfigurations for receiving both policy and resource validation requests.
   * MutatingWebhookConfigurations for receiving both policy and resource mutating requests.
+      * Note: Webhook operations per resource is dynamically set if there are match/exclude operations mentioned in the policies applied. If for a resource no operations are set in match or exclude blocks, all default operations are applied in the webhooks rules.
```
Comment (realshuting) [inline content/en/docs/Installation/_index.md:38]: This is not for mutatingwebhook only right?

Comment (chipzoller) [reply content/en/docs/Installation/_index.md:38]: Only validation? Is that what you mean?

**Comment by chipzoller** [comment]
What's the status on this one?

File: content/en/docs/Installation/_index.md (hunk 1 above)
Comment (realshuting) [reply content/en/docs/Installation/_index.md:38]: @anushkamittal2001 ?

Comment (realshuting) [reply content/en/docs/Installation/_index.md:38]: Can you clarify that only CREATE and UPDATE operations are currently configured for the mutating webhook?

**Review by realshuting — APPROVED** [review]

**Review by chipzoller — CHANGES_REQUESTED** [review]
This needs to go on the Selecting Resources page, not here.

**Review by chipzoller — APPROVED** [review]
Thanks, I'll polish this up when I start going through everything.

**Pull request merged**